iphone/Classes/TiUIProgressBar.m

```diff
@@ -21,13 +21,13 @@ -(void)initializeTiLayoutView
 }
 #endif
 
--(id)initWithStyle:(UIProgressViewStyle)style_
+-(id)initWithStyle:(UIProgressViewStyle)_style andMinimumValue:(CGFloat)_min maximumValue:(CGFloat)_max;
 {
 	if (self = [super initWithFrame:CGRectZero])
 	{
-		style = style_;
-		min = 0;
-		max = 1;
+		style = _style;
+		min = _min;
+		max = _max;
```

Comment:
I see why your using _min as the parameter cannot be named same as the property.However we could do [self setMax:max] or something else.  I don't like how  _variable for parameter looks, don't have to change to just a suggestion :P.